boards/arm/mec172xevb_assy6906/mec172xevb_assy6906.yaml

```diff
@@ -16,3 +16,9 @@ flash: 352
 supported:
   - gpio
   - pinmux
+testing:
+  ignore_tags:
+    - device
+    - pm
+    - power
+    - subys
```

Comment:
why subsys?

Reply:
@galak I remember the log was showing some PM tests were under subsys.  Anyway, I removed the ignore for subsys and the tests all pass.  Rebased with subsys ignore removed.